### PR TITLE
1. Removed usage of approvedByEditor property

### PR DESCRIPTION
**Pull request by luktar**
2. Removed assignee fields from tile details
3. Removed unused fields from tile object
4. Removed unused tiles' colours

### CODE REVIEW

File: src/paths/ManagementPanel.tsx [hunk 1]
```diff
@@ -114,54 +111,39 @@ function ManagementPanel() {
...
-              </Tooltip>
-            </Rectangle>
-          </>
-        ),
-      ),
+      tiles.map(x => (
+        <>
+          <TextPath
+            positions={[
+              [x.maxLat, x.minLon],
```
Comment (AdamSarach) [inline src/paths/ManagementPanel.tsx]: just comment: watch out for conflicts with pt-366

File: src/components/mapComponents/MapTiles.tsx [hunk 2]
```diff
@@ -24,8 +24,8 @@ const MapTiles: FC<MapTilesProps> = ({
   addReportMarker,
   isCreateReportMapMode,
 }) => {
-  const color = ({approvedByEditor}: Tile): string => {
-    return approvedByEditor ? colors.colorTileAssigned : colors.colorTileAll;
+  const color = ({assignedUserName}: Tile): string => {
```
Comment (AdamSarach) [inline src/components/mapComponents/MapTiles.tsx:27]: is really assignedUserName of type Tile?

**Review by AdamSarach — APPROVED** [review]
Some task description in Jira would be appreciated.  PT-368 name in Jira is `GTFS stops counter is always 0 in admin panel`. Had to think a bit what this PR is about

File: src/components/mapComponents/MapTiles.tsx (hunk 2 above)
Comment (luktar) [reply src/components/mapComponents/MapTiles.tsx:27]: Yes. When I've changed it started to work. 
ApprovedByEditor filed is not used anymore.

File: src/paths/ManagementPanel.tsx (hunk 1 above)
Comment (luktar) [reply src/paths/ManagementPanel.tsx]: Yes, I've fixed it in two branches to not avoid making a branch from PT-366.

**Pull request merged**